lib/backends/webgl/ops/elu.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import {Elu} from '../../../ops/elu';
+import {Tensor} from '../../../tensor';
+import {WebGLInferenceHandler} from '../inference-handler';
+import {ProgramInfo} from '../program-info';
+import {RunData} from '../program-manager';
+import {WebGLOperator} from '../webgl-operator';
+import {WebGLOperatorHelper} from '../webgl-operator-utils';
+
+export class WebGLElu extends Elu implements WebGLOperator {
+  run(inferenceHandler: WebGLInferenceHandler, inputs: Tensor[]): Tensor[] {
+    return WebGLOperatorHelper.run(this, inferenceHandler, inputs);
+  }
+  createProgramInfo(handler: WebGLInferenceHandler, inputs: Tensor[]): ProgramInfo {
+    const outputShape = inputs[0].dims.slice();
+    const shaderSource = `
+      uniform sampler2D A;
+      void main() {
+        float v = texture2D(A, TexCoords).r;
+        gl_FragColor = vec4(v >= 0.0 ? v: (exp(v) - 1.0) * ${this.alpha.toExponential()}); /* float number format */
```

Comment:
Just curious - Is there a specific reason why you cook the exponential notation of the number in the shader ? (We did not do this for leaky relu, which more or less has very similar logic, so want to know if there is value in adding this to that shader as well)

Reply:
I figured out that the default alpha for for elu is 1.0 which is formatted as integer "1" in the shader source code. This triggers a type error at the shader compiler. 

The default alpha for leaky relu is 0.1, so we didn't have this problem before. 
owever, a custom alpha of integer values such as  1.0, 2.0 could trigger a type error also.

Reply:
Thanks. Yes we did use float(value) in a lot of places to overcome the type error during shader compilation, it looks like we missed it for leaky relu and we will fix it. 

Butdo you know if there is any difference between using the float constructor (like float (value)) vs your approach ?

Reply:
I can't think of any difference.